_posts/2022-06-02-survey-2022.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+title: "Strimzi survey: we want to hear from you!"
+date: 2021-06-02
+author: paolo_patierno
+---
+
+With the help of the CNCF, we're running another survey of the Strimzi community.
+The main purpose is to hear from Strimzi users in order to know how they are using Strimzi.
+This is really useful to the maintainers to make better decisions in terms of new features and define the roadmap of the project itself.
+
+It would be great if you could take the 5–10 minutes needed to answer the 10 questions.
+
+<!--more-->
+
+## What we are interested in
+
+We are interested in knowing about your usage of Strimzi, whether it's for development, evaluation or already in production.
+In order to provide better support and backward compatibility, it is important to have an understanding of what Kubernetes versions our users are deploying Strimzi on.
+Kubernetes API changes and deprecations can have an impact on the Strimzi project as well.
+
+Similarly, it's helpful to the project to understand how quickly users adopt new Strimzi releases, and what prompts them to upgrade.
+
+One of the most recent features was the Kafka Connect Build. You can read more about it [here](https://strimzi.io/blog/2021/03/29/connector-build/).
+It allows to build a new Kafka Connect image including the connectors you need, from different sources.
+Maybe not all the users in the community know about it or others are already using it.
```

Comment:
@scholzj what about  "One of the most interesting features" ? .... I think that it's fair to say that because it simplifies users' lives with Kafka Connect images for adding connectors. How does it sound to you?

Reply:
Yeah, I think that sounds good.